core/record.go

```diff
@@ -170,7 +170,10 @@ func (t *Timetrace) LoadLatestRecord() (*Record, error) {
 		return nil, nil
 	}
 
-	dir := latestDirs[len(latestDirs)-1]
+	dir, err := t.latestNonEmptyDir(latestDirs)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I suggest returning `ErrTrackingNotStarted` to indicate that the user has no start tracking first.

```suggestion
	if err != nil {
		if errors.Is(err, ErrAllDirectoriesEmpty) {
			return nil, ErrTrackingNotStarted
		}
		return nil, err
	}
```